src/main/java/core/basesyntax/Plane.java

```diff
@@ -6,6 +6,85 @@
  * которые могут потребоваться для реализации паттерна Builder.</p>
  */
 
-public class Plane {
+public final class Plane {
```

Comment:
why did you make this class final? what does `final` keyword in the class stands for?